src/applications/vaos/services/appointment/index.js

```diff
@@ -320,15 +300,17 @@ export function isUpcomingAppointmentOrRequest(appt) {
 
 /**
  * Returns true if the given Appointment is a confirmed appointment
- * or an Express Care request
  *
  * @export
  * @param {Appointment} appt The FHIR Appointment to check
  * @returns {boolean} Whether or not the appointment is a valid upcoming
- *  appointment or Express Care request
+ *  appointment
  */
-export function isUpcomingAppointmentOrExpressCare(appt) {
-  if (CONFIRMED_APPOINTMENT_TYPES.has(appt.vaos.appointmentType)) {
+export function isUpcomingAppointment(appt) {
+  if (
+    CONFIRMED_APPOINTMENT_TYPES.has(appt.vaos.appointmentType) &&
+    !appt.vaos.isExpressCare
```

Comment:
Similar to above, this line isn't doing much, because EC requests aren't confirmed appointments